api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java

```diff
@@ -89,4 +91,13 @@ public PatientIdentifierType getPatientIdentifierTypeByNameOrUuid(String name, S
 		return sessionFactory.getCurrentSession().createCriteria(Patient.class).createAlias("names", "names")
 		        .add(ilike("names.familyName", family, MatchMode.START)).list();
 	}
+	
+	@Override
+	public List<PersonAttribute> getActiveAttributesByPatientAndAttributeTypeUuid(Patient patient,
```

Comment:
Technically, we don't need this function at all. We can just use the implementation in the `PersonDao`, which simplifies maintenance.

Reply:
@ibacher I am removing that code